packages/app/blockissuer/blockfactory/referenceprovider.go

```diff
@@ -225,7 +225,9 @@ func (r *ReferenceProvider) adjustOpinion(conflictID utxo.TransactionID, exclude
 	likedConflictID, dislikedConflictIDs := engineInstance.Consensus.ConflictResolver.AdjustOpinion(conflictID)
 
 	if likedConflictID.IsEmpty() {
-		panic("likedConflictID empty when trying to adjust opinion!")
+		return false, models.EmptyBlockID, errors.Errorf("likedConflictID empty when trying to adjust opinion for %s", conflictID)
+		// TODO: re-introduce this, as it should never happen, especially without conflict eviction
+		//panic("likedConflictID empty when trying to adjust opinion!")
```

Comment:
I think it would be best to simply return an error in this case. It should never happen, but currently we don't create the both conflicts atomically, so it may happen that we try to evaluate it before the second one is created, which could possibly cause this. We need to discuss if and how to fix that.